index.html

```diff
@@ -1066,14 +1066,10 @@ <h2>Methods</h2>
                <code>OperationError</code> and abort these steps.</li>
                <li>Set <var>stream</var>'s <a>[[\TargetReadBufferedAmount]]</a> slot to 
                <var>amount</var>.</li>
-               <li>If <var>amount</var> is greater than or equal to <var>stream</var>'s
-               <a>[[\ReadBufferedAmount]]</a> slot, set <var>stream</var>'s maximum
-               receive window to <var>amount</var>.</li>
                <li>If <var>amount</var> is less than <var>stream</var>'s
-               <a>[[\ReadBufferedAmount]]</a> slot, set <var>stream</var>'s receive
-               window to zero until <a>[[\ReadBufferedAmount]]</a> becomes less than
-               or equal to <var>amount</var> and then set the maximum receive window
-               to <var>amount</var>.</li>
+               <a>[[\ReadBufferedAmount]]</a> slot, stop increasing <var>stream</var>'s
+               MAX_STREAM_DATA offset until <a>[[\ReadBufferedAmount]]</a> becomes less than

```

Comment:
I see two options here.

1) We keep the detail of the MAX_STREAM_DATA offset usage. If we do I think it would be good to change the definition of targetReadBufferedAmount - "represents the target number of bytes in the read buffer, which enables control of back-pressure on the sender via the maximum receive window." Maybe this could be updated to "...which enables control of back-pressure on the sender via the MAX_STREAM_DATA offset."

2) Remove detail of MAX_STREAM_DATA usage. We could say "If amount is less than stream's [[ReadBufferedAmount]] slot, apply back-pressure on the sender until [[ReadBufferedAmount]] becomes less than or equal..." We could also remove a the reference to the maximum receive window from the targetReadBufferedAmount definition.

What do you think?

Reply:
How about removing the detail?

Reply:
Sounds good to me!